### PR TITLE
[E-matija < T462] implement setProperty for create module

### PR DESCRIPTION
**Pull request by mpintaric55334**
### Description

Please briefly explain the changes you made here.

### Pull request type

- [ ] Bugfix
- [ ] Algorithm/Module
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### Related issues

Delete if this PR doesn't resolve any issues. Link the issue if it does.

######################################

#### Module/Algorithm
- [ ] Core algorithm/module implementation
- [ ] [Query module](https://memgraph.com/docs/memgraph/reference-guide/query-modules) implementation
- [ ] Unit tests
- [ ] End-to-end tests
- [ ] Code documentation
- [ ] README short description
- [ ] Documentation on [memgraph/docs](https://github.com/memgraph/docs)
- [ ] Update GQLALchemy signatures in [query builder](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/graph_algorithms/query_builder.py  ) using [query module signature generator](https://github.com/memgraph/gqlalchemy/blob/main/scripts/query_module_signature_generator.py)

######################################

### CODE REVIEW

File: cpp/create_module/algorithm/create.cpp [hunk 1]
```diff
@@ -1 +1,26 @@
...
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    mgp::Graph graph = mgp::Graph(memgraph_graph);
+    const mgp::Node node_input = arguments[0].ValueNode();
+    const std::string string(arguments[1].ValueString());
+    mgp::Value value = arguments[2];
+    const int64_t id = node_input.Id().AsInt();
+    for (auto node : graph.Nodes()) {
+      if (node.Id().AsInt() == id) {
```
Comment (antoniofilipovic) [inline cpp/create_module/algorithm/create.cpp:15]: Can you check this function:
```
inline Node Graph::GetNodeById(const Id node_id) const {
  auto mgp_node = mgp::graph_get_vertex_by_id(graph_, mgp_vertex_id{.as_int = node_id.AsInt()}, memory);
  if (mgp_node == nullptr) {
    mgp::vertex_destroy(mgp_node);
    throw NotFoundException("Node with ID " + std::to_string(node_id.AsUint()) + " not found!");
  }
  auto node = Node(mgp_node);
  mgp::vertex_destroy(mgp_node);
  return node;
}
```

I am not sure why you opted for O(n) when we can have this O(log n) complexity :smile:

**Review by antoniofilipovic — CHANGES_REQUESTED** [review]
I think getting of node should be more efficient

File: cpp/create_module/algorithm/create.cpp (hunk 1 above)
Comment (mpintaric55334) [reply cpp/create_module/algorithm/create.cpp:15]: Forgot about existence of this function, will fix

**Review by antoniofilipovic — APPROVED** [review]
Looks good

**Pull request merged**